src/components/v2/values/chartValuesDiff/ChartValuesView.scss

```diff
@@ -240,7 +238,6 @@
 
         .chart-values-view__gui-form-container {
             border-bottom: 1px solid var(--N100);
-            overflow: scroll;
```

Comment:
is it intentional?

Reply:
Yeah, have added overflow css with `chart-values-view__gui-form-container` in ChartValueGUIView component